vignettes/wip/dm-db.Rmd

```diff
@@ -95,7 +129,12 @@ my_dm_keys %>%
   dm_add_tbl(trans)
 ``````
 
-- Reminder: all operations are transient unless stored in a new variable
+For tables from other sources or the local environment `[copy_to()](missing_link)` from the {dplyr} package is used.
```

Comment:
Style/grammar question: does the additional "from" help or hurt the sentence?

Code will be auto-linked on the website.

```suggestion
For tables from other sources or from the local environment `dplyr::copy_to()` is used.
```